jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1424,7 +1427,7 @@
     (let (avs)
       (setq avs
             (list
-              (send *baxter* :approach-from-downside arm)
+              (send *baxter* :midpose-hold-opposite-hand-object-downside arm)
```

Comment:
For longterm use, I think we should use some planning algorithms with checking collisions with tote or some other objects,
as we checked norm distance between IK result and `:avoid-shelf-pose-X` in `:move-arm-body->bin` with multiple predefined poses.
Using a single hardcoded-pose works in very limited situations.